spec/Entity/BillingDataSpec.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace spec\Sylius\InvoicingPlugin\Entity;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Component\Resource\Model\ResourceInterface;
+use Sylius\InvoicingPlugin\Entity\BillingDataInterface;
+
+final class BillingDataSpec extends ObjectBehavior
+{
+    function let(): void
+    {
+        $this->beConstructedWith(
+            'John',
+            'Doe',
+            'US',
+            'Foo Street 3/44',
+            'New York',
+            '24154'
+        );
+    }
+
+    function it_implements_billing_data_interface(): void
+    {
+        $this->shouldImplement(BillingDataInterface::class);
+    }
+
+    function it_implements_resource_interface(): void
+    {
+        $this->shouldImplement(ResourceInterface::class);
+    }
+
+    function it_has_proper_billing_data(): void
+    {
+        $this->beConstructedWith(
```

Comment:
Since you don't need to change constructor parameters in several tests, why don't you put this in let() function?